GentApp/Views/Branch/EditBranchPage.xaml.cs

```diff
@@ -61,7 +61,7 @@ public sealed partial class EditBranchPage : Page {
 				isValid = false;
 			}
 			if ( isValid ) {
-				SimpleIoc.Default.GetInstance<CompanyViewModel>().EditBranch(
+				await SimpleIoc.Default.GetInstance<CompanyViewModel>().EditBranch(
```

Comment:
heeft da veel nut met die await's erbij en async in methode?

Reply:
ja da zorgt ervoor dat hij alles in de juiste volgorde doet en bv niet eerst company ophaalt en dan maar de update stuurt.